triggers/service/src/main/resources/com/daml/lf/engine/trigger/db/migration/oracle/V1__Init.sql.sha256

```diff
@@ -1 +1 @@
-6dc2d7d667ab3052662754853f3aeaaa8323ea6f14184312eef5c3ca56a90f1c
+9f5baba232ea6606dcd0c995207e6ec88bfdf495afd897a2e595f6c96365d763
```

Comment:
AFAIK These are only used by [our tests](https://github.com/digital-asset/daml/blob/a8d703b9960d70158493234a6dfcfd1be3abb014/libs-scala/flyway-testing/src/main/scala/flyway/AbstractImmutableMigrationsSpec.scala) to avoid unwanted changes of migration steps. In this case, I don't see a way around changing these. The change is such that as long as the table prefix is empty, the migrations are unchanged.